frame/fast-unstake/src/benchmarking.rs

```diff
@@ -74,7 +74,7 @@ fn setup_staking<T: Config>(v: u32, until: EraIndex) {
 		.collect::<Vec<_>>();
 
 	for era in 0..=until {
-		let others = (0..T::MaxBackersPerValidator::get())
+		let others = (0..T::Staking::max_exposure_page_size())
```

Comment:
Is this benchmark only considering a page size of 1? If we go multi page, with `MaxExposurePageCount` > 1, then would we need a double iterator for iterating the actual `page` count and `others` within them?

Reply:
Clarified that a call only deals with 1 page so multi-page not needed.